<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserCenter</class>
 <widget class="QDialog" name="UserCenter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="CenterListWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <item>
    <property name="text">
     <string>修改密码</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>修改邮箱</string>
    </property>
   </item>
  </widget>
  <widget class="QStackedWidget" name="CentreStackWidget">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>30</y>
     <width>241</width>
     <height>161</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="CentreOPLabel">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>20</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>原密码：</string>
     </property>
    </widget>
    <widget class="QLabel" name="CentreNPLabel">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>52</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>新密码：</string>
     </property>
    </widget>
    <widget class="QLabel" name="CentreNPQLabel">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>83</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>确认密码：l</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CentrePOkButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="CentreOPLineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="CentreNPLineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="CentreNPQLineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="CentreEWarning">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="CenterOEMLabel">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>20</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>原邮箱：</string>
     </property>
    </widget>
    <widget class="QLabel" name="CentreNEMLabel">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>52</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>新邮箱：</string>
     </property>
    </widget>
    <widget class="QLabel" name="CentreNEMQLabel">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>83</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>确认邮箱：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="CentreOEMLineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="CentreNEMLineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="CentreNEMQLineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="CentreEButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>130</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
    <widget class="QLabel" name="CentreEWarnLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>134</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>108</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>请输入你的qq邮箱</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="CentreLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; font-style:italic;&quot;&gt;个人中心&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="CentreNLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>5</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>用户：</string>
   </property>
  </widget>
  <widget class="QLabel" name="CentreNameLabel">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>5</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="CentreNoButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
